src/base/PackableStream.h

```diff
@@ -9,6 +9,7 @@
 #ifndef SQUID_SRC_BASE_PACKABLESTREAM_H
 #define SQUID_SRC_BASE_PACKABLESTREAM_H
 
+#include "base/IoManip.h"
```

Comment:
To avoid duplicating discussions, we should revisit this _after_ src/base/AsyncJobCalls.h concern is addressed.